shared/src/components/Header/Header.tsx

```diff
@@ -289,17 +289,22 @@ export const Header = ({
               </a>
             </li>
           )}
-          <li
-            className={classNames("p-navigation__link", {
-              "is-selected": location.pathname.startsWith(
-                generateURL("/account/prefs", false, appendNewBase)
-              ),
-            })}
-            role="menuitem"
-          >
-            {authUser &&
-              generateLink({ url: "/account/prefs", label: authUser.username })}
-          </li>
+          {completedIntro && (
+            <li
+              className={classNames("p-navigation__link", {
+                "is-selected": location.pathname.startsWith(
+                  generateURL("/account/prefs", false, appendNewBase)
+                ),
+              })}
+              role="menuitem"
+            >
+              {authUser &&
+                generateLink({
+                  label: authUser.username,
+                  url: "/account/prefs",
+                })}
+            </li>
+          )}
```

Comment:
I changed this to hide the user prefs link if the intro had not been completed, otherwise it was possible to navigate to user prefs with no way to get back to the intro (apart from the browser back button)

Reply:
Nice, I think it would even just redirect you back to the intro if you clicked it and hadn't finished the intro.